modules/upgrade/pages/db-migration-12.adoc

```diff
@@ -0,0 +1,123 @@
+[[db-migration-12]]
+= Database Migration from Version 10 to 12
+
+This section covers upgrading the PostgreSQL database from version{nbsp}10 to version{nbsp}12.
+If you are already using PostgreSQL 12, you do not need to perform this migration.
+If you are using an older version, such as version 9.6, see xref:upgrade:db-migration.adoc[].
+
+If you want to upgrade to the latest {productname} version, you must be using PostgreSQL version 10 or 12.
+You migrate to PostgreSQL version 12 after you upgraded your {productname} Server to version{nbsp}4.1.
+
+
+
+////
+== Background Information
+
+
+* {productname} 3.1.4, SLES 12 SP3, Postgres 9.6
+* {productname} 3.1.11, SLES 12 SP3, Postgres 9.6
+* {productname} 3.2.10, SLES 12 SP3, Postgres 9.6
+* {productname} 3.2.10, SLES 12 SP3, Postgres 10
+* {productname} 3.2.10, SLES 12 SP4, Postgres 10
+* {productname} 4.0.0, SLES 15 SP1, Postgres 10
+* {productname} 4.1.0, SLES 15 SP2, Postgres 12
+
+////
+
+
+
+
+[[db-migration-12-prepare]]
+== Prepare to Upgrade
+
+Before you begin the upgrade, prepare your existing {productname} Server and create a database backup.
+
+PostgreSQL stores data at [path]``/var/lib/pgsql/data/``.
+
+.Procedure: Preparing to Upgrade
+
+. Check the active PostgreSQL version:
++
+----
+psql --version
+----
++
+If you are using PostgreSQL{nbsp}10, you can upgrade to PostgreSQL{nbsp}12.
++
+If you are already using PostgreSQL version 12, you do not need to perform this migration.
+. Check the active smdba version:
++
+----
+rpm -q smdba
+----
++
+// CHECKIT!
++
+PostgreSQL{nbsp}10 requires ``smdba`` version 1.6.2 or later.
```

Comment:
I'm not sure but we night to update smdba. I think Michael is already aware of that.

Reply:
I did it already. If i remember correcty we require not smdba 1.7.4. The current version in Head

Reply:
1.7.4 is correct.